LastWork(newWork)/css/p_newevaluation.css

```diff
@@ -0,0 +1,155 @@
+.image{
+	width: 150px;
+	height: 30px;
+	margin-left: 50px;
+}
+
+.header{
+	width: 100%;
+	height: 55px;
+	border-bottom: 1px solid;
+	display: inline-flex;
+	align-items: center;
+	box-shadow: 0px 3px 10px grey;
+}
+
+.header_div{
+	display: flex;
+	width: 1200px;
+	margin: 0 auto;
+}
+
+.body_section{
+	display: flex;
+	width: 1200px;
+	height: 100%;
+	margin: 0 auto;
+}
+
+.eval_button{
+	text-decoration: none;
+	padding: 0px 50px;
+	float:left;
+	margin-bottom: 0px;
+}
+
+.neweval_button{
+	border-bottom: 10px solid;
+	border-bottom-color: #03a8f3;
+	text-decoration: none;
+	padding: 0px 50px;
+	float: left;
+	margin-bottom: 0px;
+}
+
+.logout{
+	float: right;
+	text-decoration: none;
+	margin-right: 50px;
+}
+
+.footer_section{
+	width: 100%; 
+	height: 100px; 
+	margin-top: 40px;
+	background-color: #DFDFDF;
+	position: absolute;
+}
+
+footer{
+	width: 1200px;
+	margin: 0 auto;
+}
+
+footer p{
+	margin: 40px 0px;
+	text-align: right;
+	width: 100%;
+}
+
+
+.textbox {
+	width: 30%;
+	margin-top: 5%;
+	margin-right: 0.5%;
+	padding: 1%;
+	border: none;
+}
+
+.textbox::placeholder{
+	border-bottom: 1px solid lightgrey;
+	color: lightgrey;
+}
+
+.textbox:focus{
+	outline: none;
+	border-bottom: 1px solid #DFDFDF;
+}
+
+.boxTitle {
+	color: #3090C7;
+	margin-left: 35px;
+}
+
+ul{
+	list-style-type: none;
+	padding-left: 15px;
+}
+
+li{
+	float: left;
+	width: 30%;
+	margin: 5px 25px 15px 0px;
+	font-size: 20px;
+}
+
+fieldset{
+	margin-top: 50px;
+}
+
+textarea{
+	height: 100px;
+	resize: none;
+	width: 100%;
+	box-sizing: border-box;;
+}
+
+select{
+	float: right;
+	width: 140px;
+	height: 40px;
+}
+
+legend{
+	color: #3090C7;
+	font-size: 22px;
+	font-weight: bold;
+	padding: 0px 20px;
+}
+
+#radio_select{
```

Comment:
Loose the id selectors. Loose the DOM tag names selectors with the exceptions of the styles that should be applied site wide.